### PR TITLE
Fixed location for current executed assembly in GetAzureResourceGroup Cmdlet.

### PR DESCRIPTION
**Pull request by hovsepm**

### CODE REVIEW

File: src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeCloudExceptionTests.cs [hunk 1]
```diff
@@ -20,9 +20,12 @@ namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
 {
     public class ComputeCloudExceptionTests
     {
+        ServiceManagemenet.Common.Models.XunitTracingInterceptor logger;
```
Comment (fearthecowboy) [inline src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeCloudExceptionTests.cs]: ouch! are you aware that "ServiceManagemenet" is misspelled? 

I think it's been that way for a while :cry:

**Comment by fearthecowboy** [comment]
Looks good to me :+1:

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/867/

**Comment by hyonholee** [comment]
LGTM, Thanks.

**Comment by hovsepm** [comment]
covered in #2352

**Pull request closed without merging**